shared/util/memoize.js

```diff
@@ -1,7 +1,15 @@
 // @flow
 import memoize from 'memoize-one'
 
+const memoize1Obj = (f: any) =>
+  memoize(
+    f,
+    // $FlowIssue - the type of memoize1 is wonky
+    (a, b) => Object.keys(a).length === Object.keys(b).length && !Object.keys(a).some(k => a[k] !== b[k])
```

Comment:
why not just use shallowequal?

Reply:
maybe prefer this to be named memoizeShallow

Reply:
I forgot about that lib